README.md

```diff
@@ -1,35 +1,174 @@
 
+
+You can run FAIR Data Point without the client if you need the API only. If you want a user interface for browsing the metadata and administration of the FAIR Data Point metadata and users, you can deploy FAIR Data Point together with this client.
+
+FDP Client works as a proxy before the FAIR Data Point itself. It decides which request should pass through to the FDP (e.g., API calls) and which should be handled by the client (requests from browsers). Therefore, you no longer need to have FAIR Data Point exposed publicly.
+
+
 ## Deployment
 
-FAIR Data Point Client is distributed as a Docker image. It is meant to be run together with the FAIR Data Point. The configuration needed is API url (without the trailing slash) of the FDP server as a `API_URL` ENV variable and host name of FDP server for proxying requests when running the Docker image.
+FAIR Data Point Client is distributed as a Docker image. It runs together with the FAIR Data Point.
 
+**Configuration**
 
-You can use [Docker Compose](https://docs.docker.com/compose/) to run FDP and FDP client together:
+| ENV Variable | Description |
+| ---- | --- |
+| FDP_HOST | A hostname of the FAIR Data Point (within the Docker network). |
+| PUBLIC_PATH | Use only if FDP is not running at the root, you need to specify the URL. For example, if you run FDP at https://example.com/fairdatapoint, PUBLIC_PATH should be `/fairdatapoint`. |
+
+**Example**
+
+Here is an example [Docker Compose](https://docs.docker.com/compose/) configuration to run FDP and FDP client together:
 
 ```yaml
 version: '3'
 services:
     server:
         image: faridata/fairdatapoint
-        # ... server configuration
+        # ... FDP configuration
 
     client:
         image: fairdata/fairdatapoint-client
         ports:
             - 80:80
         environment:
-            - API_URL=http://api.example.com
             - FDP_HOST=server  # using hostname within the Docker network
 ```
 
-> Tip: It is a good idea to run FDP and FDP client behind a reverse proxy with SSL certificates.
+> Tip: It is recommended to run FDP and FDP client behind a reverse proxy with SSL certificates. See further in the docs examples how to do that.
+
+## Customizations
+
+You can customize the look and feel of FDP Client using [SCSS](https://sass-lang.com). There are three files you can mount to `/src/scss/custom`. If there are any changes in these files, the styles will be regenerated when FDP Client starts.
+
+**_variables.scss**
+
+A lot of values related to styles are defined as variables. The easiest way to customize the FDP Client is to define new values for these variables. To do so, you create a file called `_variables.scss` where you define the values that you want to change.
+
+Here is an example of changing the primary color.
+
+```scss
+// _variables.scss
+
+$color-primary: #087d63;
+```
+
+Have a look in `src/scss/_variables.scss` to see all the variables you can change.
```

Comment:
Maybe add a link to the file, so users can easily inspect the file from their browser?